pkg/controller/plan/kubevirt.go

```diff
@@ -54,7 +54,7 @@ import (
 // Annotations
 const (
 	// Transfer network annotation (value=network-attachment-definition name)
-	AnnDefaultNetwork = "v1.multus-cni.io/default-network"
+	AnnDefaultNetwork = "k8s.v1.cni.cncf.io/networks"
```

Comment:
Q: do we also want to change the variable name ? since it's not the "default" network annotation anymore ?